src/Modules/Render/ViewScene.cc

```diff
@@ -229,6 +229,7 @@ void ViewScene::syncMeshComponentFlags(const std::string& connectedModuleId, Mod
 
 void ViewScene::execute()
 {
+  /*
```

Comment:
@Haydelj Was this "to be worked on later"? Not that it worked well before...

Reply:
Yeah as of now if that code is uncommented viewScene will infinitely execute. Since it was already broken I figured we could figure out why this happens when we get around to making it work correctly

Reply:
I can take a look at it now if you would prefer though

Reply:
It can wait